docs/logging.md

```
@@ -59,39 +60,38 @@ This document outlines the logging standards and best practices for Ramen.
-    ```
+  ```go
+  s.log.Info("reconcile end", "time spent", time.Since(start))
+  ```
 
-    This will log in the following format, providing clear and structured context:
+  This will log in the following format, providing clear and structured context:
 
-    ```
-    2024-01-31T14:27:12.726-0500  INFO    pvrgl   reconcile end   {"name": "protectedvrglist-0", "rid": "1e09b0fb-687b-4100-9ab1-a52ba899b37b", "rv": "322", "time spent": 6ms}
-    ```
+  ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)